bitmovin/resources/models/encodings/encoding.py

```diff
@@ -75,6 +78,23 @@ def encoderVersion(self, new_version):
             raise InvalidTypeError(
                 'Invalid type {} for encoderVersion: must be either str or EncoderVersion!'.format(type(new_version)))
 
+    @property
+    def labels(self):
+        return self._labels
+
+    @labels.setter
+    def labels(self, new_labels):
+        if new_labels is None:
```

Comment:
Please set `self._labels` to `None` in this case

Reply:
fixed and pushed to PR